### PR TITLE
define `+cwt`, `s/eat-[cj]wt/eat+[cj]wt`

### LINKED ISSUE #14: use `+jwt` SSS?
IANA's [Structured Syntax Suffixes registry](https://www.iana.org/assignments/media-type-structured-suffix/media-type-structured-suffix.xhtml) has `+jwt`, defined in [RFC8417](https://www.rfc-editor.org/rfc/rfc8417.html#section-7.2) to be _"used indicate that the media type is encoded as a JWT."_

It looks like `application/eat+jwt` could be used instead of `application/eat-jwt`.

### PR DESCRIPTION
**Pull request by thomas-fossati**
Fix #14

### CODE REVIEW

File: materials/ietf-115-rats-eat-media-types.md [hunk 1]
```diff
@@ -70,23 +70,36 @@ autoscale: true
 
 # Next Steps (find the bug, win a beer!)
 
+[.column]
+
 ```golang
-waitChan := make(chan bool)
+func nextSteps(earlyAllocation bool) {
+	waitChan := make(chan bool)
```
Comment (henkbirkholz) [inline materials/ietf-115-rats-eat-media-types.md:77]:
```suggestion
        waitChan := make(chan bool)
```

File: materials/ietf-115-rats-eat-media-types.md [hunk 2]
```diff
@@ -70,23 +70,36 @@ autoscale: true
...
 
-if earlyAllocation == true {
-	go IANA(waitChan)
-}
+	if earlyAllocation {
+		go IANA(waitChan)
+	}
```
Comment (henkbirkholz) [inline materials/ietf-115-rats-eat-media-types.md:81]:
```suggestion
        if earlyAllocation {
                go IANA(waitChan)
        }
```

File: materials/ietf-115-rats-eat-media-types.md [hunk 3]
```diff
@@ -70,23 +70,36 @@ autoscale: true
...
 
-for _, issue := range openIssues {
-	if issue.isBlocking {
-		sendPR(issue)
+	for _, issue := range openIssues {
+		if issue.isBlocking {
+			sendPR(issue)
+		}
```
Comment (henkbirkholz) [inline materials/ietf-115-rats-eat-media-types.md:86]:
```suggestion
        for _, issue := range openIssues {
                if issue.isBlocking {
                        sendPR(issue)
                }
```

File: materials/ietf-115-rats-eat-media-types.md [hunk 4]
```diff
@@ -70,23 +70,36 @@ autoscale: true
...
 	}
+
+	<-waitChan
+
+	submit(0x02)
```
Comment (henkbirkholz) [inline materials/ietf-115-rats-eat-media-types.md:91]:
```suggestion
        <-waitChan

        submit(0x02)
```

File: materials/ietf-115-rats-eat-media-types.md [hunk 5]
```diff
@@ -70,23 +70,36 @@ autoscale: true
...
 }
+```
 
-<-waitChan
+[.column]
 
-submit(02)
+```golang
+func IANA(waitChan chan bool) {
+	fmt.Println("do it!")
```
Comment (henkbirkholz) [inline materials/ietf-115-rats-eat-media-types.md:99]:
```suggestion
        fmt.Println("do it!")
```

**Comment by thomas-fossati** [comment]
:trollface: 🤣

File: materials/ietf-115-rats-eat-media-types.md (hunk 1 above)
Comment (thomas-fossati) [reply materials/ietf-115-rats-eat-media-types.md:77]: :trollface:

File: materials/ietf-115-rats-eat-media-types.md (hunk 2 above)
Comment (thomas-fossati) [reply materials/ietf-115-rats-eat-media-types.md:81]: :trollface:

File: materials/ietf-115-rats-eat-media-types.md [hunk 6]
```diff
@@ -70,23 +70,36 @@ autoscale: true
...
 
-```
+	// actually do it
```
Comment (henkbirkholz) [inline materials/ietf-115-rats-eat-media-types.md:101]:
```suggestion
        // actually do it
```

File: materials/ietf-115-rats-eat-media-types.md (hunk 3 above)
Comment (thomas-fossati) [reply materials/ietf-115-rats-eat-media-types.md:86]: :trollface:

File: materials/ietf-115-rats-eat-media-types.md (hunk 4 above)
Comment (thomas-fossati) [reply materials/ietf-115-rats-eat-media-types.md:91]: :trollface:

File: materials/ietf-115-rats-eat-media-types.md (hunk 5 above)
Comment (thomas-fossati) [reply materials/ietf-115-rats-eat-media-types.md:99]: :trollface:

File: materials/ietf-115-rats-eat-media-types.md [hunk 7]
```diff
@@ -70,23 +70,36 @@ autoscale: true
...
 
- 
+	waitChan <- true
```
Comment (henkbirkholz) [inline materials/ietf-115-rats-eat-media-types.md:103]:
```suggestion
        waitChan <- true
```

File: materials/ietf-115-rats-eat-media-types.md (hunk 6 above)
Comment (thomas-fossati) [reply materials/ietf-115-rats-eat-media-types.md:101]: :trollface:

File: materials/ietf-115-rats-eat-media-types.md (hunk 7 above)
Comment (thomas-fossati) [reply materials/ietf-115-rats-eat-media-types.md:103]: :trollface:

**Review by henkbirkholz — APPROVED** [review]
plus is better than minus

**Comment by thomas-fossati** [comment]
> plus is better than minus

yes, a wise old man once said that "plus is more"

**Pull request merged**